src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -480,31 +53,59 @@
     }
 
     &:hover {
-      color: var(--navbar-active);
+      color: var(--);
     }
   }
 
   .topRowWrapper {
     display: flex;
-    grid-area: top;
-    gap: var(--space-sm);
-    z-index: 30;
+    grid-area: header;
+    align-items: center;
+    justify-content: space-between;
+    gap: auto;
+    z-index: 1;
+    width: 100%;
 
     & > * {
       filter: drop-shadow(4px 4px 8px rgba(0, 0, 0, 0.25));
     }
 
     .backButton {
-      margin-inline-end: auto;
+      display: flex;
+      justify-content: flex-start;
+      gap: var(--space-sm);
+      align-self: start;
     }
   }
 
+  .topRowWapperInner {
+    display: flex;
+    grid-area: header2;
```

Comment:
there's no grid area with the name `header2`

Reply:
I will remove it, thanks for the catch